journal-temp/modernized/02-chapter-2.adoc

```diff
@@ -177,9 +184,14 @@ for my giving him the title of T. Waldgrave, in my letter to him,
 according to his own subscription to his warrant of commitment, as is hereafter shown.
 A poor pretence for detaining me fifteen months in prison!
 
-The substance of a letter which I sent to Thomas Waldgrave,
-after he committed me to Bury jail.
+[.offset]
+_The substance of a letter which I sent to Thomas Waldgrave,
```

Comment:
I wasn't sure how to handle these things here, and there are several of them. They are italicized in the ODT. I tried different things and this is what I settled on. Maybe there is some cut and dry way to do them that I have forgotten?? 😄

Reply:
So, scratch that, I think I found the lint error while looking at the PR. If there are more, lemme know!